packages/theme/src/components/ThemeProvider.tsx

```diff
@@ -67,6 +69,12 @@ const ThemeProvider = ({
     getDefaultTheme(override),
   );
   const { key, mode } = theme;
+  let computedMode: ThemeMode = mode;
```

Comment:
I think that if this is truly beta then the default mode on L45 should be light, not pref.  The user should have to opt into it, I think.

Unfortunately, all our existing users proably already have "pref" in their localStorage.

Reply:
Spoke with @substrae and the fact that we likely have already stored "pref" and the beta flag is just us being super careful then we should be good to merge as is.

Users can always set light mode if they don't like our dark mode too.. I guess 🤷‍♀️ 😅